basalt/sdk/monitorsdk.py

```
@@ -1,12 +1,16 @@
-from typing import Dict, Optional, Any
+from typing import Dict, Optional, Any, Tuple
 
 from ..utils.protocols import IApi, ILogger
-from ..utils.dtos import TraceParams, GenerationParams, LogParams
-
+from ..ressources.monitor.trace_types import TraceParams
+from ..ressources.monitor.experiment_types import ExperimentParams
+from ..ressources.monitor.generation_types import GenerationParams, Generation
+from ..ressources.monitor.log_types import LogParams, Log
 from ..objects.trace import Trace
 from ..objects.generation import Generation
```

Ruff (0.11.9)

Redefinition of unused `Generation` from line 6

(F811)